src/tests/functional/shared_test_classes/src/base/snippets_test_utils.cpp

```diff
@@ -60,6 +60,12 @@ void SnippetsTestsCommon::setInferenceType(ov::element::Type type) {
     configuration.emplace(ov::hint::inference_precision(type));
 }
 
+void SnippetsTestsCommon::setDefaultSnippetsMode() {
+    if (!configuration.count("SNIPPETS_MODE")) {
+        configuration.insert({"SNIPPETS_MODE", "IGNORE_CALLBACK"});
+    }
+}
+
```

Comment:
Good idea to encapsulate configuration update into separate method 👍🏼 

To be honest, it's a little bit curios to see that default mode is `IGNORE_CALLBACK`. Also default mode is usually set by "default" - it means that you don't need to set default value then😄 

Could we rename this method please? What's about `setTestSnippetsMode` or `setIgnoreCallbackMode`?

cc @v-Golubev